src/Link_TEST.cc

```diff
@@ -203,3 +205,61 @@ TEST(DOMLink, InvalidInertia)
   EXPECT_DOUBLE_EQ(3.4, link.Inertial().MassMatrix().OffDiagonalMoments().Z());
   EXPECT_FALSE(link.Inertial().MassMatrix().IsValid());
 }
+
+/////////////////////////////////////////////////
+TEST(DOMLink, ValidInertialPoseRelTo)
+{
+  std::ostringstream stream;
+  stream << "<?xml version=\"1.0\"?>"
+         << "<sdf version='1.7'>"
+         << "  <model name='A'>"
+         << "    <link name='B'>"
+         << "      <inertial>"
+         << "        <pose relative_to=''>0 0 0 0 0 0</pose>"
+         << "      </inertial>"
+         << "    </link>"
+         << "  </model>"
+         << "</sdf>";
+
+  sdf::SDFPtr sdf(new sdf::SDF());
```

Comment:
Since the main public API for reading SDFormat strings these days is `sdf::Root::LoadSdfString`, I'd prefer if you used that for this test as well as the `InvalidInertialPoseRelTo` test.

Reply:
78bd7730848e8c1c5fe9fd7d601003f97609333b